File: 110algoritmos/exercicios015/ex015-Go/ex015.go
package main

import "fmt"

func main() {
	var dias float32
	var km float32

	fmt.Println("Quandos dias alugados? ")
	fmt.Scan(&dias)
	fmt.Println("Quantos Km rodados? ")
	fmt.Scan(&km)

	total := (dias * 60) + (km * 0.15)

	fmt.Printf("\nO total a pagar é R$ %.2f", total)

}

File: 110algoritmos/exercicios015/ex015-Elixir/ex015.exs
d = IO.gets("Quantos dias alugados? ")
{dias, _} = Float.parse(d)

k = IO.gets("Quantos km rodados? ")
{km, _} = Float.parse(k)

total = (dias* 60.0) + (km * 0.15)

IO.puts("O total a pagar eh R$ #{total}")
